README.md

```diff
@@ -1,18 +1,51 @@
-Hosted website Link https://guru-jii.herokuapp.com/
+## Technology Stack used:
+<img src="https://img.shields.io/badge/reactjs%20-61DAFB.svg?&style=for-the-badge&logo=react&logoColor=blue"/> <img src="https://img.shields.io/badge/github%20-%23121011.svg?&style=for-the-badge&logo=github&logoColor=white"/> <img src="https://img.shields.io/badge/heroku%20-%23430098.svg?&style=for-the-badge&logo=heroku&logoColor=white"/> <img src="https://img.shields.io/badge/express.js%20-%23404d59.svg?&style=for-the-badge"/> <img src ="https://img.shields.io/badge/MongoDB-%234ea94b.svg?&style=for-the-badge&logo=mongodb&logoColor=white"/>
 
-Want to contribute? Follow the steps below to set it up locally.
+- **Frontend**: ReactJS
+- **Backend**: Node.js, Express.js
+- **IDE**: VS Code
+- **Version Control**: Git and GitHub
+- **Database**: MongoDb
+- **Hosting**: Heroku
 
-Step 1: Fork and Clone the repo.<br />
-Step 2: run `cd backend yarn install` to install backend dependencies.<br />
-step 3: run `cd .. yarn install` to install frontend dependencies.<br />
-Step 4: run `yarn dev`. This should launch the website on `localhost:3000`.
+## Getting Started 🚀 
+### Want to contribute? Follow the steps below to set it up locally.
+
+```
+# Clone this repository
+$ git clone https://github.com/your-username/MERN-Gurujii-dev.git
+
+# Go into the repository
+$ cd MERN-Gurujii-dev
+
+# Install backend dependencies
+$ cd backend yarn install
+
+# Install frontend dependencies
+$ cd .. yarn install
+
+# Run website
+$ yarn dev
+```
+This should launch the website on `localhost:3000`.
 
 We have used proxy so all the requests made to backend from frontend will be served at
 `localhost:5000` at which our backend is running.
 
-Great! Now you are good to go. Open/pick an issue, discuss your idea, make the changes in your feature branch and create your first pull request in this repo.
+## Hosted website Link 📦 
+
+Click Here to view the deployment!
+ 
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://guru-jii.herokuapp.com/)
```

Comment:
Can the button only show the project name?

Reply:
According to you what should label of button contain? 
I will work on other mentioned feedbacks.

Reply:
pending!

Feel free to add any other detail or anything you think would be good for the participants to contribute.

Reply:
I have used already created heroku button svg .It can't be edited. What else I can do?

Reply:
can we change deploy to deployed? it would look like deployed to heroku